helm.go

```diff
@@ -164,6 +165,10 @@ This will not alter the charts in your workspace.
 			ArgsUsage: "[chart-name...]",
 			Action: func(c *cli.Context) {
 				minArgs(c, 1, "uninstall")
+				// TODO: remove global kubectl.Runner
```

Comment:
:+1: on removing it. we talked about passing around some kind of context instead of setting a global here. I'm in favor of doing that, as it will allow for us to write tests that are more explicit as to what's going on. I'd also consider that more in line with the "go way"